DevEdu.API/DevEdu.API/Models/InputModels/GroupInputModel.cs

```diff
@@ -1,14 +1,23 @@
-﻿using System;
+﻿using DevEdu.DAL.Enums;
+using System;
 using System.ComponentModel.DataAnnotations;
+using static DevEdu.API.Common.ValidationMessage;
 
 namespace DevEdu.API.Models.InputModels
 {
     public class GroupInputModel
     {
+        [Required(ErrorMessage = NameRequired)]
+        public string Name { get; set; }
         [Required]
         public int CourseId { get; set; }
+        [Required(ErrorMessage = GroupStatusIdRequired)]
+        public GroupStatus GroupStatusId { get; set; }
+        [Required(ErrorMessage = DateRequired)]
```

Comment:
StartDate must be a string in the input model
with our custom validation attribute